code/modules/mob/living/living_verbs.dm

```diff
@@ -243,9 +235,7 @@
 				for(var/mob/O in viewers(CM))
 					O.show_message(text("<span class='danger'>[] is trying to break [LC]!</span>", CM), 1)
 				spawn(0)
-					if(do_after(CM, 50, FALSE, 5, BUSY_ICON_HOSTILE))
```

Comment:
vIs the removal of the icon intended here?

Reply:
idem

Reply:
You shouldn't change functionality here